articles/flutter-vs-kotlin/index.md

```diff
@@ -0,0 +1,87 @@
+
+In this article, we will be comparing Flutter & Kotlin as cross-platform development languages. The primary goal of this article is to help developers distinguish the best cross-development platform for them, depending on their needs, and skills.
+
+### Performance
+Kotlin and Flutter, are superb choices when it comes to mobile app development, both with their own distinctions. Kotlin can compile the code and customize it to the best needs of its user as well as the native apps.
+
+Flutter has a reloading functionality, that gives the developer a choice of selection if all the different elements are performing well.
+
+Developers describe Flutter as one of the most popular cross-platform mobile framework. Kotlin, however, is described more as a statistical-type programming language.
+
+### Language
+One of the main advantages of Kotlin is that its programming language is quite similar to Java. Therefore developers who have worked with Java in the past will find it much easier to work with Kotlin. 
+On the other hand, Flutter uses Dart from Google. Due to the fact that Dart is a client-optimized programming language, it may offer an easier option for developers who have strong knowledge of C languages. It is easier to understand and operate due to fast compiling like C languages.
+
+### Price and Testing
+Kotlin and Flutter are both free, but the real difference occurs in testing support. While Kotlin tends to support backend development, Flutter works on extensive testing of features.
+
+### UI Support
+Kotlin as a multi-platform development tool has a unique distinction of lending a helping hand to UI developers and allows them to make a truly customizable app without restrictions. On the other hand, developers working with Flutter can take full advantage of widgets, which also helps in crafting customized solutions.
+
+### Pros of Kotlin
+Kotlin is an SDK, so it is not necessary to shift the complete app to the platform. Due to its similarities with Java, developers find it easier to get a handle on it and use a multi-frame to design a cross-platform app. Since the UI creation is not restricted, developers have the freedom to work on any aspect of the app.
+
+### Cons of Kotlin
+There are very few built-in widgets and libraries available for developers, as Kotlin is still new in the market and still growing. Once the date-time library arrives, the situation will improve. The development cost for Kotlin is quite high as well, and it will eventually be the desired option for the developers in the long run.
```

Comment:
What would the built-in widgets and libraries be used for in Kotlin? Why is development cost high? Last sentence is speculative.